src/drivers/drv_led.h

```diff
@@ -57,5 +57,8 @@
 // set the queue size to the number of LED's, so that each led can be controlled individually
 static const int LED_UORB_QUEUE_LENGTH = BOARD_MAX_LEDS;
 
-
+#if defined (__PX4_POSIX_OCPOC)
```

Comment:
In the case of board specific changes. So that they can be reused by other boards, they should come from the board_config.h 
So in  board_config.h  add:
```#define BOARD_RGBLED0_PATH "/dev/i2c-1"```

Then here:
```
#if defined(BOARD_RGBLED0_PATH)
#  define RGBLED0_DEVICE_PATH BOARD_RGBLED0_PATH
#else
 #  define RGBLED0_DEVICE_PATH "/dev/rgbled0"
#endif
```